src/chi-vue/src/components/data-table/data-table.scss

```diff
@@ -31,3 +31,75 @@
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+@mixin respond-to($media, $only: false) {
+  @if $media == xs {
+    @media only screen and (max-width: $sm-breakpoint - 1) {
+      @content;
+    }
+  } @else if $media == sm {
+    @if $only {
+      @media only screen and (min-width: $sm-breakpoint) and (max-width: $md-breakpoint - 1) {
+        @content;
+      }
+    } @else {
+      @media only screen and (min-width: $sm-breakpoint) {
+        @content;
+      }
+    }
+  } @else if $media == md {
+    @if $only == true {
+      @media only screen and (min-width: $md-breakpoint) and (max-width: $lg-breakpoint - 1) {
+        @content;
+      }
+    } @else {
+      @media only screen and (min-width: $md-breakpoint) {
+        @content;
+      }
+    }
+  } @else if $media == lg {
+    @if $only {
+      @media only screen and (min-width: $lg-breakpoint) and (max-width: $xl-breakpoint - 1) {
+        @content;
+      }
+    } @else {
+      @media only screen and (min-width: $lg-breakpoint) {
+        @content;
+      }
+    }
+  } @else if $media == xl {
+    @media only screen and (min-width: $xl-breakpoint) {
+      @content;
+    }
+  }
+}
+
+$md-breakpoint: 768px;
+$color-border-base-dark: #d0d4d9;
+$font-weight-semi-bold: 600;
```

Comment:
Set TODO comment to remove needed code when all µApps use Chi version higher than v5.0